modules/configuring-firewall.adoc

```diff
@@ -46,6 +46,10 @@ There are no special configuration considerations for services running on only c
 |443, 80
 |The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`
 
+|`redhat-developer.github.io`
```

Comment:
I'm sorry, I didn't see any requests to `redhat-developer.github.io` when trying to browse Databases in Developer Console, here is my screenshot, could you please let me know how did you find this? 
it is sending requests to `https://<console_route>/api/devfile/samples/?registry=https://registry.devfile.io`
![Screen Shot 2022-09-14 at 16 08 59](https://user-images.githubusercontent.com/12692381/190098511-10c9a05c-b97e-4b73-a25b-2c98a333c3e7.png)